pages/dashboard/ServiceInvitations.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+import getConfig from 'next/config';
+
+import DisplayInvitations from './DisplayInvitations';
+
+/**
+ * COMPONENT 'ApplicationSection'
```

Comment:
Please clean up and add some text for what this component is doing... I think it's worth it here.